### PR TITLE
Update enum usage

### PR DESCRIPTION
**Pull request by WeihanLi**
Thought there's no need to specify `int`, is there some concern?

### CODE REVIEW

**Comment by Kahbazi** [comment]
@WeihanLi FYI this is the previous conversation about the issue : https://github.com/microsoft/reverse-proxy/pull/501/files#r517487666

**Comment by MihaZupan** [comment]
I explicitly added the backing type here as changing it could break binary compatibility of our EventSource events between versions (enums may not always serialize as 4 bytes)

**Comment by karelz** [comment]
Perhaps we could add comment there why we explicitly added the backing type?

File: src/ReverseProxy.TelemetryConsumption/Forwarder/ForwarderStage.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace Yarp.ReverseProxy.Telemetry.Consumption
     /// <summary>
     /// Stages of forwarding a request.
     /// </summary>
-    public enum ForwarderStage : int
+    public enum ForwarderStage
```
Comment (WeihanLi) [inline src/ReverseProxy.TelemetryConsumption/Forwarder/ForwarderStage.cs:9]: Seemed this enum definition is the same with `src/ReverseProxy/Forwarder/ForwarderStage.cs`, maybe we could share the same one?

Comment (MihaZupan) [reply src/ReverseProxy.TelemetryConsumption/Forwarder/ForwarderStage.cs:9]: They are almost the same, but one is public while the other is internal.
Adding conditional compilation for just this file seemed like more effort than it's worth.

**Pull request closed without merging**